js/notifications.js

```diff
@@ -46,34 +46,65 @@ weechat.factory('notifications', ['$rootScope', '$log', 'models', 'settings', fu
                 });
             });
         } else {
-            var notification = new Notification(title, {
-                body: body,
-                icon: 'assets/img/favicon.png'
-            });
-
-            // Save notification, so we can close all outstanding ones when disconnecting
-            notification.id = notifications.length;
-            notifications.push(notification);
+            
+            //If it detects the Windows object, that is, running on a Hosted Web App container, use the UWP toasts
+            if (typeof Windows !== 'undefined' && 
+                typeof Windows.UI !== 'undefined' && 
+                typeof Windows.UI.Notifications !== 'undefined') {
+                    
+                    var winNotifications = Windows.UI.Notifications;
```

Comment:
Can you fix the indentation here? It shouldn't be two blocks. AIso I find the blank lines between the declarations make this unnecessarily hard to read